src/crud.js

```diff
@@ -0,0 +1,39 @@
+const inputName = document.querySelector('.input-name');
+const inputScore = document.querySelector('.input-score');
+const ul = document.querySelector('.names-container');
+
+export const postUser = async () => {
+  const res = await fetch(
+    'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/NGhCX4R3qCILaTqPLKM0/scores/',
+    {
+      method: 'POST',
+      body: JSON.stringify({
+        user: inputName.value,
+        score: inputScore.value,
+      }),
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+    }
+  );
+  const data = await res.json();
+  console.log(data);
```

Comment:
- Optional
Consider removing console.log statements.

Reply:
Hi Code Reviewer,
Thanks for taking out time to review my code. In your review above, you said no linters error (GitHub workflows help me correct every error). I prefer using Github workflows to check for line errors, and I passed all of them including the javascript linter error (eslint). I really don't understand why this should affect my review.
Cheers